src/components/ErrorBoundary/ErrorBoundary.js

```diff
@@ -27,18 +33,60 @@ class ErrorBoundary extends React.Component<Props, State> {
         onError: noop,
     };
 
-    componentDidCatch(error: Error): void {
+    constructor(props: Props) {
+        super(props);
+        this.updateOnError(props.onError);
+    }
+
+    componentDidUpdate() {
+        this.updateOnError(this.props.onError);
+    }
+
+    componentDidCatch(error: Error, info: Object): void {
```

Comment:
Minor, but I believe the second param, `errorInfo`, has a flow-type defined in React.

Reply:
I took a look, doesnt seem to be exposed. Maybe upgrading flow will fix it